device/globals/action_menu.gd

```diff
@@ -14,7 +14,7 @@ func start(p_target):
 		target = p_target
 		target.connect("visibility_changed", self, "target_visibility_changed")
 
-	var scale = Globals.get("platform/action_menu_scale")
+	var scale = ProjectSettings.get("platform/action_menu_scale")
```

Comment:
Does it work?

According to the documentation the correct function seems to be `get_setting()`, but there's inconsistency between the example and the listed functions (`set()` vs `set_setting()`), so I'm not sure what's right.

Reply:
If you want, I can squash/fixup `get_setting`, but I went with `get` in the second commit for consistency. Because the Godot editor still autocompletes `get`, it can't be all wrong.

Reply:
Ok, that's probably still good then. I see that `get()` is also used pretty much everywhere in Escoria, so that sounds like a good call to me.

Reply:
Yeah, I think it would be good to squash the commits, but might as well wait and see if one of the maintainers have any suggested changes.

Reply:
If `get(...)` is most commonly used, I usually prefer to stick to the fewest surprises and most consistency in my usual programming. Then it's fine to have a separate branch to change them all to `get_setting(...)` later if that becomes relevant. Not my project and I'm fine either way with this, just my two cents for whomever wants to comment.

Reply:
I agree.

Reply:
`get()` is actually a `Node` method. `ProjectSettings::get_setting()` actually directly calls `get()` as you can see here: https://github.com/godotengine/godot/blob/9f479f096cb50fa8d1215e68c262f110116114ef/core/project_settings.cpp#L854
`get()` won't disappear so there's no problem using `get()` for now. However, if at some point `get_setting()` evolves and performs other actions, then it would be cleaner to change every `ProjectSettings.get()` in Escoria in favor of `ProjectSettings.get_setting()`.

@mjtorn If you are ok, you can perform the change on the whole Escoria directory with this command (under Linux):
`find ./ -type f -exec sed -i -e 's/ProjectSettings.get\(/ProjectSettings.get_setting\(/g' {} \;`
That should do the trick :)

Reply:
Pushed such a commit, thanks for the feedback! :)